src/UseCases/office.js

```diff
@@ -0,0 +1,36 @@
+const Office = require('../Models/office');
+const bcrypt = require('../Lib/bcrypt');
+
+function getOffices(){
+    return Office.find({})
+}
+
+function getById(idOffice){
+    return Office.findById(idOffice)
+}
+
+function create(dataOffice){
+    const {name, address, email, password, rfc, clients, residents, projects} = dataOffice
+
+    const officeFound = await Office.findOne({office: office})
+    if(officeFound) throw new Error("Not permision to create, this office already exist");
+
+    const passwordEncrypt = await bcrypt.hash(password)
+    return Office.create({name, address, email, password: passwordEncrypt, rfc, clients, residents, projects}) 
```

Comment:
```suggestion
    return Office.create({...dataOffice password: passwordEncrypt}) 
```

Con spread operator se ahorran el pasar todas las propiedades al crear la office 🙃.  @KristomeisterDev

Reply:
muy cierto Fer, era lo que hacíamos con populate en la practica, no se me ocurrió pero muchas gracias no me vuelve a fallar